spacy_llm/models/hf/base.py

```diff
@@ -27,12 +27,46 @@ def __init__(
         inference_config (Dict[Any, Any]): HF config for model run.
         """
         self._name = name if self.hf_account in name else f"{self.hf_account}/{name}"
-        self._config_init, self._config_run = self.compile_default_configs()
+        default_cfg_init, default_cfg_run = self.compile_default_configs()
+        self._config_init, self._config_run = default_cfg_init, default_cfg_run
+
         if config_init:
             self._config_init = {**self._config_init, **config_init}
         if config_run:
             self._config_run = {**self._config_run, **config_run}
```

Comment:
Still wrapping my head around how this actually works. Shouldn't we only add keys from the `default_X` dictionaries that aren't already in either `config_init` or `config_run`, instead of just plainly merging these?

Reply:
The entries in the second dict overwrite the entries in the first dict - so e. g. `{**{"a": 1, "b": 2}, **{"a": 3, "c": 4}} == {'a': 3, 'b': 2, 'd': 4}`. Meaning that if we have something in our default config, user-specified entries override those from the default config. This should be identical to only adding values from the default config that aren't in the user-provided config.